test/meson.build

```
@@ -0,0 +1,24 @@
+hba_test = executable('hba_test',
+                      'hba_test.c',
+                      '../src/hba.c',
+                      '../src/util.c',
+                      libusual_sources,
+                      config_h,
+                      include_directories: ['..', '../include', '../lib', '../uthash/src'],
+                      dependencies: [libevent, openssl],
```

If systemd is available then libusual fails to link without this, due to logging.c calling `sd_journal_print`

```suggestion
                      dependencies: [libevent, openssl, systemd],
```